### PR TITLE
added r-castor recipe

### PR DESCRIPTION
**Pull request by gavinmdouglas**
Recipe for [castor R package](https://cran.r-project.org/web/packages/castor/index.html) built based on instructions from https://github.com/bgruening/conda_r_skeleton_helper.

This recipe was built with the ```bash run.sh``` command.

Please note that I am not the developer of castor.

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
You may want to add more r maintainers to this if possible.

**Comment by gavinmdouglas** [comment]
Hi @mariusvniekerk, are there particular maintainers of conda-forge you know of that I could add?

**Comment by bgruening** [comment]
@gavinmdouglas in the skeleton helper that you referenced above are a bunch of r maintainers :)

**Comment by gavinmdouglas** [comment]
Thanks for clarifying @bgruening! I re-added the names that were originally listed as maintainers.

File: recipes/r-castor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+about:
+  home: https://CRAN.R-project.org/package=castor
+  license: GPL (>= 2)
+  summary: Efficient tree manipulations for large phylogenies, including pruning, rerooting,
+    calculation of most-recent common ancestors, calculating distances from the tree
+    root and calculating pairwise distances. Calculation of phylogenetic signal and
+    mean trait depth (trait conservatism), ancestral state reconstruction and hidden
+    character prediction of discrete characters. Simulating and fitting models of trait
+    evolution, and generating random trees using birth-death models. Dating trees, comparing
+    two trees, and reading/writing trees to a file. Citation
```
Comment (cbrueffer) [inline recipes/r-castor/meta.yaml]: The summary is cut off; looks good apart from this!

Comment (gavinmdouglas) [reply recipes/r-castor/meta.yaml]: Great thanks! I added the rest of the summary (it was cut-off due to a ":" character I think).

**Pull request merged**